reclaimer/sound/ogg_new.py

```diff
@@ -0,0 +1,82 @@
+import struct
+from Encoder import OGGEncoder
+from supyr_struct.defs.audio.wav import wav_def
+
+"""
+    OGG Encoder:
+        Creation: 06/15/2020
+        Last Edit: 06/18/2020 (Zatarita)
+"""
+
+class OGG():
+    filepath = None
+    mode = ""
+
+    def __init__(self, filepath, mode):
+        self.filepath = filepath
+        self.mode = mode
+
+    def __enter__(self):
+        if self.mode == "w":
+            return Encoder(self.filepath)
+
+    def __exit__(self, type, value, traceback):
+        return
+
+class Encoder():
+    bitspersample = 0
+    channelcount = 0
+    samplerate = 0
+    quality = 1
+    log = False
+
+    filestream = None
+
+    Data = []
```

Comment:
Defining this this way will make it look like a class when we later use it. I'd prefer snake_case for this.